Makefile

```diff
@@ -96,22 +97,30 @@ work: $(GOBIN)
 # format section
 ############################################################
 
+fmt-dependencies:
+	$(call go-get-tool,$(PWD)/bin/gci,github.com/daixiang0/gci@v0.2.9)
+	$(call go-get-tool,$(PWD)/bin/gofumpt,mvdan.cc/gofumpt@v0.2.0)
+
 # All available format: format-go format-protos format-python
 # Default value will run all formats, override these make target with your requirements:
 #    eg: fmt: format-go format-protos
-fmt: # format-go format-protos format-python
-	go fmt ./...
+fmt: fmt-dependencies
+	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gofmt -s -w
+	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gofumpt -l -w
 
 ############################################################
 # check section
 ############################################################
 
 check: lint
 
+lint-dependencies:
+	$(call go-get-tool,$(PWD)/bin/golangci-lint,github.com/golangci/golangci-lint/cmd/golangci-lint@v1.41.1)
```

Comment:
Let me know if we have decided to use the same fixed golangci-lint version in all repos

Reply:
@ChunxiAlexLuo I'd like to since the newer version adds additional linting rules that we didn't discuss as a team.